pwnlib/tubes/tube.py

```diff
@@ -520,20 +561,23 @@ def recvline(self, keepends=True, timeout=default):
             b'trailing data'
             >>> t.recvline() # doctest: +ELLIPSIS
```

Comment:
Should also work w/o ELLIPSIS

Reply:
Worked for me locally as well, but most (all?) tests in this file have this. I'll remove it everywhere and we'll check if it passes.